reanalysis/vep/query.py

```diff
@@ -0,0 +1,62 @@
+"""
+Hail Query function to parse JSON VEP results.
+"""
+
+import hail as hl
+
+
+def vep_json_to_ht(vep_results_paths, out_path):
+    """
+    Parse results from VEP with annotations formatted in JSON,
+    and write into a Hail Table.
+    """
+    # Defining schema inside the function, so we can submit
+    # the function to the Batch Job:
+    schema = hl.dtype(
+        'struct{assembly_name:str,allele_string:str,ancestral:str,'
```

Comment:
[nitpicking] I'd probably keep schema in the original format, and transform it in runtime. The reason is that I stole it from https://github.com/hail-is/hail/blob/cc0a051740f4de08408e6a2094ffcb1c3158ee9c/hail/python/hailtop/hailctl/hdinsight/resources/vep-GRCh38.json#L19, so it would be just a lit bit easier to keep in sync

Reply:
Yeah, I swapped it out to avoid doing the string manipulation at runtime (actually in response to the JSON parsing error which was really just a file truncation), but the replace operations are so trivial in the context of the whole process that I'll revert it to the original.